Gopkg.toml

```diff
@@ -70,6 +70,19 @@
  name = "github.com/containerd/containerd"
  revision = "f05672357f56f26751a521175c5a96fc21fa8603"
 
+[[constraint]]
+ name = "github.com/go-openapi/runtime"
```

Comment:
Nit: This is a TOML file, so I think it requires you to use 2 spaces to indent the entries under `constraints`.

Reply:
Looks like we did the same for containerd too. Let fix both to make it consistent